simpleclient_spring_web/src/main/java/io/prometheus/client/spring/web/MethodTimer.java

```diff
@@ -0,0 +1,64 @@
+package io.prometheus.client.spring.web;
+
+import io.prometheus.client.Summary;
+import org.apache.commons.lang3.StringUtils;
+import org.aspectj.lang.ProceedingJoinPoint;
+import org.aspectj.lang.annotation.Around;
+import org.aspectj.lang.annotation.Aspect;
+import org.springframework.stereotype.Component;
+import org.springframework.web.bind.annotation.ControllerAdvice;
+
+import java.util.HashMap;
+
+/**
+ * This class automatically times (via aspectj) the execution of methods by their signature, if it's been enabled via {@link EnablePrometheusTiming}
+ * and in methods annotated (or within classes annotated) with {@link PrometheusTimeMethods}
+ *
+ * @author Andrew Stuart <andrew.stuart2@gmail.com>
+ */
+@Aspect
+@ControllerAdvice
```

Comment:
My experience was that without this annotation, Spring ignored the `@PrometheusMethodTiming` annotation anywhere inside controllers. This annotation enabled that, which to me is the most applicable use case for high-level monitoring.